salt/cloud.sls

```diff
@@ -59,6 +60,8 @@ salt-cloud-{{ cloud_section }}:
     - template: jinja
     - makedirs: True
     - exclude_pat: _*
+    - context:
+          salt_settings: {{ salt_settings|json }}
```

Comment:
Fix indentation here - it's not consistent.

Reply:
LGTM but just this change is needed.

Reply:
Good catch ;)